### PR TITLE
Fix build issues for iOS.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

File: talk/owt/sdk/base/peerconnectiondependencyfactory.h [hunk 1]
```diff
@@ -73,9 +73,9 @@ class PeerConnectionDependencyFactory : public rtc::RefCountInterface {
   CreatePeerConnectionOnCurrentThread(
       const webrtc::PeerConnectionInterface::RTCConfiguration& config,
       webrtc::PeerConnectionObserver* observer);
-#if defined(WEBRTC_WIN) || defined(WEBRTC_LINUX)
-  rtc::scoped_refptr<webrtc::AudioDeviceModule> CreateCustomizedAudioDeviceModuleOnCurrentThread();
-#endif
+  rtc::scoped_refptr<webrtc::AudioDeviceModule>
```
Comment (taste1981) [inline talk/owt/sdk/base/peerconnectiondependencyfactory.h:76]: why will you need this for iOS?

File: talk/owt/sdk/base/peerconnectiondependencyfactory.cc [hunk 2]
```diff
@@ -201,7 +201,6 @@ void PeerConnectionDependencyFactory::
   } else {
     encoder_factory = webrtc::CreateBuiltinVideoEncoderFactory();
   }
-#endif
 
   if (GlobalConfiguration::GetCustomizedVideoDecoderEnabled()) {
```
Comment (taste1981) [inline talk/owt/sdk/base/peerconnectiondependencyfactory.cc:205]: Please confirm you've customized video decoder support for iOS?

File: talk/owt/sdk/base/peerconnectiondependencyfactory.h (hunk 1 above)
Comment (jianjunz) [reply talk/owt/sdk/base/peerconnectiondependencyfactory.h:76]: iOS supports custom audio input as well.

File: talk/owt/sdk/base/peerconnectiondependencyfactory.cc (hunk 2 above)
Comment (jianjunz) [reply talk/owt/sdk/base/peerconnectiondependencyfactory.cc:203]: Custom video decoder is not supported on iOS. But it should work as GetCustomizedVideoDecoderEnabled always returns false on iOS.

**Comment by jianjunz** [comment]
This change depends on https://github.com/open-webrtc-toolkit/owt-deps-webrtc/pull/180, which fixes iOS build issues for deps-webrtc. Please review this one as well. Thanks.

**Review by taste1981 — APPROVED** [review]

**Pull request merged**